internal/pkg/earpointer/state.go

```diff
@@ -106,30 +86,30 @@ func (state *State) AllPartitionReps() ReferenceSet {
 	return reps
 }
 
-// GetPartitionRep gets the partition representative of reference "aref"
+// Representative gets the partition representative of reference "aref"
 // ("aref" must belong to this state).
-func (state *State) GetPartitionRep(aref Reference) Reference {
+func (state *State) Representative(aref Reference) Reference {
 	failureCallback := func(aref Reference) Reference {
-		log.Fatalf("GetPartitionRep: Reference [%s] not found in state", aref)
+		log.Fatalf("Representative: Reference [%s] not found in state", aref)
 		return nil
 	}
 	return state.lookupPartitionRep(aref, failureCallback)
 }
 
-// GetPartitionFieldMap gets the partition-level field map for partition
+// PartitionFieldMap gets the partition-level field map for partition
 // representative "aref". The result maps a field (ConstTermHandle type)
 // to a reference that is a partition representative.
 // Note "aref" must be a partition representative.
-func (state *State) GetPartitionFieldMap(aref Reference) ReferenceFieldMap {
-	return state.getPartitionInfo(aref).fieldRefs
+func (state *State) PartitionFieldMap(aref Reference) FieldMap {
```

Comment:
Is it possible to split the type of Reference and Representative apart? This seems like a win for clarity if it is doable.

Reply:
Representative is a special Reference. While it is possible to separate them, doing that may bring other complexity, e.g. by introducing an interface for Reference and Representative so that they can have the same APIs.